conformance_pack/s3.sp

```diff
@@ -260,3 +260,483 @@ control "s3_bucket_object_logging_enabled" {
+        else b.title || ' publicly readable.'
+      end reason
+      -- Additional Dimensions
+      ${local.tag_dimensions_sql}
+      ${replace(local.common_dimensions_qualifier_sql, "__QUALIFIER__", "b.")}
+    from
+      aws_s3_bucket as b
+      left join data as d on b.name = d.name;
+  EOQ
+}
+
+query "s3_bucket_restrict_public_write_access" {
+  sql = <<-EOQ
+    with data as (
+      select
+        distinct name
+      from
+        aws_s3_bucket,
+        jsonb_array_elements(acl -> 'Grants') as grants
+      where
+        grants -> 'Grantee' ->> 'URI' = 'http://acs.amazonaws.com/groups/global/AllUsers'
+        and (
+          grants ->> 'Permission' = 'FULL_CONTROL'
+          or grants ->> 'Permission' = 'WRITE_ACP'
+        )
+      )
+    select
+      -- Required Columns
+      b.arn as resource,
+      case
+        when d.name is null then 'ok'
+        else 'alarm'
+      end status,
+      case
+        when d.name is null then b.title || ' not publicly writable.'
+        else b.title || ' publicly writable.'
+      end reason
+      -- Additional Dimensions
+      ${local.tag_dimensions_sql}
+      ${replace(local.common_dimensions_qualifier_sql, "__QUALIFIER__", "b.")}
+    from
+      aws_s3_bucket as b
+      left join data as d on b.name = d.name;
+  EOQ
+}
+
+query "s3_bucket_versioning_enabled" {
+  sql = <<-EOQ
+    select
+      -- Required Columns
+      arn as resource,
+      case
+        when versioning_enabled then 'ok'
+        else 'alarm'
+      end status,
+      case
+        when versioning_enabled then name || ' versioning enabled.'
+        else name || ' versioning disabled.'
+      end reason
+      -- Additional Dimensions
+      ${local.tag_dimensions_sql}
+      ${local.common_dimensions_sql}
+    from
+      aws_s3_bucket;
+  EOQ
+}
+
+query "s3_public_access_block_account" {
+  sql = <<-EOQ
+    select
+      -- Required Columns
+      'arn' || ':' || 'aws' || ':::' || account_id as resource,
+      case
+        when block_public_acls
+          and block_public_policy
+          and ignore_public_acls
+          and restrict_public_buckets
+          then 'ok'
+        else 'alarm'
+      end as status,
+      case
+        when block_public_acls
+          and block_public_policy
+          and ignore_public_acls
+          and restrict_public_buckets
+          then 'Account level public access blocks enabled.'
+        else 'Account level public access not enabled for: ' ||
+          concat_ws(', ',
+            case when not (block_public_acls ) then 'block_public_acls' end,
+            case when not (block_public_policy) then 'block_public_policy' end,
+            case when not (ignore_public_acls ) then 'ignore_public_acls' end,
+            case when not (restrict_public_buckets) then 'restrict_public_buckets' end
+          ) || '.'
+      end as reason
+      -- Additional Dimensions
+      ${local.tag_dimensions_sql}
+      ${local.common_dimensions_sql}
```

Comment:
```suggestion
      ${local.common_dimensions_qualifier_global_sql}
```

Reply:
I think it will be `${local.common_dimensions_global_sql}`

Reply:
@madhushreeray30 yes it will be `${local.common_dimensions_global_sql}` , please update the same.